specs/makeStyles.md

```diff
@@ -307,21 +352,6 @@ const theme = {
 }
 ```
 
-### Always matching matcher
-
-What is the best signature for a `makeStyles` call with a single style object which always matches?
-
-```js
-makeStyles([[null, { background: 'red ' }]]);
-makeStyles([{ background: 'red' }]);
-makeStyles({ background: 'red' });
-makeOverrides({ background: 'red' });
-```
-
-### `makeStyles` per component or per slot
```

Comment:
Now it can be defined in a single call:

```tsx
makeStyles({
  root: { color: "red" },
  content: { color: "red" },
  icon: { color: "red" }
  // etc.
});
```